src/modules/Core/style-settings.scss

```
@@ -1842,9 +1842,12 @@ settings:
     id: anp-bold-custom
     title: Bold Color
     type: class-select
-    allowEmpty: true
+    allowEmpty: false
     default: anp-bold-red
     options:
+      - 
+        label: None
+        value: anp-bold-none
```

It might be better to add no class at all instead of a redundant class.
```suggestion
        value: none
```

---

resolved in 853d345d0d6531abf47eae87759ecfe0dce900ed